src/AbstractContainerCommandLoader.php

```diff
@@ -69,6 +76,10 @@ private function createCommand(string $name): Command
             throw ConfigurationException::withInvalidMappedCommandClass($name, $class);
         }
 
-        return new $class();
+        /** @psalm-suppress MixedMethodCall */
```

Comment:
This becomes a more precise error (although still to be suppressed) if `$name` is documented to be a `class-string<Command>` in the method definition

Reply:
Where are these various pseudo-types documented? I've been having trouble finding them (I was trying to look up some of them I'd seen in automatic-releases, such as "non-empy-string" and "non-empty-array", but not finding any listing anywhere).

Reply:
https://psalm.dev/docs/annotating_code/type_syntax/scalar_types/

Reply:
Thanks, @boesing ! That page is nowhere in the navigation, which might be why searches do not unearth it (no links!).

Bookmarking!